mm2src/mm2_test_helpers/dummy_files/iris_test_history.json

```diff
@@ -18,7 +18,7 @@
             "type": "Tendermint",
             "coin": "IRIS-TEST",
             "amount": "0.022145",
-            "gas_limit": 100000
+            "gas_limit": 125000
```

Comment:
this is because https://github.com/KomodoPlatform/atomicDEX-API/blob/a6f9a0ea383d6f87f2157bb6598a76eb7fa10071/mm2src/coins/my_tx_history_v2.rs#L454